tests/suites/tenant/summary/ObjectSummary.ts

```diff
@@ -181,8 +195,61 @@ export class ObjectSummary {
         return true;
     }
 
+    async getTreeItem(text: string) {
+        // Ensure tree is ready for the search
+        await this.isTreeVisible();
+        await this.isTreeLoaded();
+
+        const itemLocator = this.treeRows.filter({hasText: text}).first();
+
+        // Default timeout is too big for such case
+        const timeout = 500;
```

Comment:
I didn't use default timeouts here because they are too big for such case. In this method we assume, that the tree is present and already loaded with `isTreeVisible` and `isTreeLoaded`. We spend little time to check elements, since in the worst scenario there will be 10 loops - with default timeout (`VISIBILITY_TIMEOUT`) it may be more than 100 seconds for one function call (in my version max is about 7 seconds)